src/core/blockchain/common/account.ts

```diff
@@ -0,0 +1,25 @@
+import { Blockchain } from '../types';
+import { BigNumber } from 'bignumber.js';
+
+export const convertUnit = (
+    blockchain: Blockchain,
+    value: BigNumber,
+    fromUnit,
+    toUnit,
+    info
```

Comment:
this param is not needed, you already have `blockchain`, just import `BLOCKCHAIN_INFO` and `info = BLOCKCHAIN_INFO[blockchain]`

Reply:
Problem with this is with circular import. problem discovered when running tests